io/teca_cf_layout_manager.cxx

```diff
@@ -293,18 +367,44 @@ int teca_cf_layout_manager::define(const teca_metadata &md_in,
         }
     }
 
+    std::string coord_array_name_in_file[4];
```

Comment:
Could you please add some comments around/in this block?  It looks like this block looking for the presence of '/' to (1) determine if a coordinate is in a group and (2) extract the coordinate name itself if so.  This might not be obvious to others (or even to me) later on though

Reply:
That code doesn't exist anymore. I refactored things during a later commit, but it seems you are still seeing the old version. I added more comments in the function I created for this.